docs/content/preview/architecture/transactions/concurrency-control.md

```diff
@@ -291,7 +291,7 @@ Each retry will use a newer snapshot of the database in anticipation that the co
 
 Note that the retries will not be performed in case the amount of data to be sent from YSQL to the client proxy exceeds the TServer gflag `ysql_output_buffer_size`.
 
-## Wait-on-Conflict [[Beta]](/preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag) {#wait-on-conflict}
+## Wait-on-Conflict [[Tech Preview]](/preview/releases/versioning/#feature-availability) {#wait-on-conflict}
```

Comment:
Messes up right nav
Maybe instead put in a note below heading

Few paragraphs below, rewrite xref "Refer [this](../read-committed/#interaction-with-concurrency-control) for more information." as "Refer to [name of heading] for more information."